KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Network.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) CoSMo Software Consulting Pte. Ltd. - All Rights Reserved
+ */
+
+package io.cosmosoftware.kite.instrumentation;
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+import io.cosmosoftware.kite.report.Status;
+
+import javax.json.JsonObject;
+
+public class Network {
```

Comment:
rename to Profile

I think @sajidcosmo need to make this an OGM entity to match with the model Profile

and merge NWCommand into this class